.github/workflows/ci.yml

```diff
@@ -0,0 +1,19 @@
+name: CI
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 22
+
+      - run: |
```

Comment:
```suggestion
      - name: Install and Build
        run: |
```

Reply:
I thought it showed up on the UI, but just checked: https://github.com/safe-global/safe-singleton-factory/actions/runs/16570594661 and `build` is actually fine.